docker/web/src/main/docker/nginx/default.conf

```diff
@@ -18,6 +18,13 @@ server {
         add_header Cache-Control 'must-revalidate, max-age=0';
     }
 
+    # Route /ide/whatever to serve ide/index.html but preserve user-supplied URL for js to process
+    location ~ ^/ide/ {
+        try_files $uri $uri/ /ide/index.html =404;
+        #expires -1;
+        #add_header Cache-Control 'must-revalidate, max-age=0';
```

Comment:
Don't think these should be commented out?

Reply:
Seems to work w/ these uncommented out.

Reply:
Even w/ these uncommented tho, a page using this feature like https://jxn-test-changes.demo.deephaven.app/ide/notebook/03%20Kafka%20Stream%20vs%20Append.md still gives me `304` cached status...